docs/etrinh1.md

```diff
@@ -0,0 +1,7 @@
+# ETRINH1 Bio
+
+** Name: **   GitHub Student etrinh1
+
+** Role: **   Student
+
+** Objective: **  Learn GitHub
```

Comment:
Thanks for your feedback.  I don't have anything specific; however, since we will start using GitHub to collaborate and manage our development and changes, I want to try to do it the right intended way.